### PR TITLE
importccl: fix MYSQLOUTFILE edge cases

### PR DESCRIPTION
**Pull request by madelynnblue**

### CODE REVIEW

File: pkg/ccl/importccl/read_import_mysqlout.go [hunk 1]
```diff
@@ -17,11 +17,13 @@ import (
...
 	"github.com/cockroachdb/cockroach/pkg/sql/sqlbase"
+	"github.com/cockroachdb/cockroach/pkg/util/ctxgroup"
 )
 
 type mysqloutfileReader struct {
-	csvInputReader
-	opts roachpb.MySQLOutfileOptions
+	conv     rowConverter
+	opts     roachpb.MySQLOutfileOptions
+	debugRow func(tree.Datums)
```
Comment (dt) [inline pkg/ccl/importccl/read_import_mysqlout.go]: is this used?

File: pkg/ccl/importccl/import_stmt_test.go [hunk 2]
```diff
@@ -206,13 +260,20 @@ d
...
-				t.Errorf("unexpected: %v", err)
+				t.Fatalf("unexpected: %v", err)
 			}
 			for query, res := range tc.query {
 				sqlDB.CheckQueryResults(t, query, res)
 			}
 		})
 	}
+
+	t.Run("mysqlout multpile", func(t *testing.T) {
```
Comment (dt) [inline pkg/ccl/importccl/import_stmt_test.go]: nit: multiple

**Review by dt — APPROVED** [review]

File: pkg/ccl/importccl/read_import_mysqlout.go (hunk 1 above)
Comment (mjibson) [reply pkg/ccl/importccl/read_import_mysqlout.go]: was during debugging. removed.

**Pull request merged**